app/Lib/lang.php

```diff
@@ -1831,6 +1837,7 @@
   'fd.plugin.warn' => 'Once a new %1$s has been created, the Plugin cannot be changed',
   'fd.prov.group' =>  'Provisioning Group',
   'fd.prov.group.desc' => 'If set, only provision members of the specified group to this target',
+  'fd.prov.max_retry.desc' => 'After the provisioning fails it will automatically retry for XX times. If set to 0 (zero) it will retry for infinite times',
```

Comment:
instead of XX this should say something like "... retry up to the specified number of times. If set to 0 (zero) it will retry indefinitely"

Reply:
done